project.py

```diff
@@ -336,29 +342,63 @@ def area_code_prefixes() -> list:
     return output_list
 
 
+def show_summary_data(title, data):
+    data_table = {title: data}
+    df = pd.DataFrame(data_table).sort_values(by=title, ascending=False)
+    print(f'=== {title} ===')
+    # If there are more than 10 values show only the top-10
+    print(df.head(10))

```

Comment:
The Top 10 format used in the pie charts was a great idea since it allows for a lot of pertinent information to be displayed without overloading an end user with information. It's also good that the total was included as well so that other statistical information such as percentages could be derived from it and users will know that other datapoints exist.